google-cloud-tools-plugin/build.gradle

```diff
@@ -62,6 +62,8 @@ dependencies {
         exclude group: 'org.yaml', module: 'snakeyaml'
     }
     compile files('lib/google-api-services-source.jar')
+
+    compile 'com.google.guava:guava:20.0'
```

Comment:
GCS is on 20 still? *sigh*

Reply:
it is: https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/pom.xml#L139